nb_templates/make_release_plots.ipynb

```diff
@@ -59,7 +59,7 @@
      "output_type": "stream",
      "text": [
       "INFO: Working on J1643-1224 [unknown]\n",
-      "INFO: Par file created: 2022-03-05T02:00:26.071758 [timing_analysis.timingconfiguration]\n"
+      "INFO: Par file created: 2022-03-05T02:00:26.071758 [pint_pal.timingconfiguration]\n"
```

Comment:
It looks like a version of this notebook was checked in without removing clearing all output. I think it would be worth fixing that in this PR.

Reply:
Fixed it!